### PR TITLE
Add new versions of Holybro KakuteH7 and H7Mini

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-holybro-kakuteh7-v2.rst [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+============
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+   -  SERIAL0 -> USB
+   -  SERIAL1 -> UART1 (Telem1) DMA-enabled
+   -  SERIAL2 -> not available externally, internally connected to currently unsupported BT-LE module
+   -  SERIAL3 -> UART3 (SmartAudio VTX protocol by default)
+   -  SERIAL4 -> UART4 (GPS)D MA-enabled
```
Comment (andyp1per) [inline common/source/docs/common-holybro-kakuteh7-v2.rst:138]: typo

File: common/source/docs/common-holybro-kakuteh7-v2.rst [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+Servo Output Mapping
+====================
+
+The PWM outputs are marked M1-M8 in the above pinouts. The corresponding servo outputs are:
+
+    - M1 -> Servo Output 4
+    - M2 -> Servo Output 1
+    - M3 -> Servo Output 2
+    - M4 -> Servo Output 3
+    - M5 -> Servo Output 5
```
Comment (andyp1per) [inline common/source/docs/common-holybro-kakuteh7-v2.rst:157]: Is this correct?

**Review by andyp1per — CHANGES_REQUESTED** [review]

File: common/source/docs/common-autopilots.rst [hunk 3]
```diff
@@ -59,8 +59,8 @@ Closed hardware
     Holybro Kakute F4 Mini* <common-holybro-kakutef4-mini>
     Holybro Kakute F7 AIO* <common-holybro-kakutef7aio>
     Holybro Kakute F7 Mini* (only V1 and V2 are compatible) <common-holybro-kakutef7mini>
-    Holybro Kakute H7-BT <common-holybro-kakuteh7.rst>
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-autopilots.rst:62]: per Holybro...no longer manufactured soon tob e removed from website

File: common/source/docs/common-holybro-kakuteh7-v2.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-holybro-kakuteh7-v2.rst:138]: fixed

File: common/source/docs/common-holybro-kakuteh7-v2.rst [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+
+The SERIAL7 port (UART7) is normally for ESC telemetry, and has an R7 pin on
+both of the ESC connectors.
+
+Any UART may be re-tasked by changing its protocol parameter.
+
+Servo Output Mapping
+====================
+
+The PWM outputs are marked M1-M8 in the above pinouts. The corresponding servo outputs are:
```
[the remark this answers is not in the document]
Comment (Hwurzburg) [reply common/source/docs/common-holybro-kakuteh7-v2.rst:151]: will it? did not check that....probably  should not be like this...this was just whatever was in the H7-BT doc....probably it was wrong...default is QUAD+ with motors assigned 1-4 to outputs 1-4...probably should fix this and the discontinued H7-BT doc, unless the hwdef.dat really has PB0 going to the M4 board output,etc...can you check?
if we wanted to default the BF/X we should do so in the default.params and in the hwdef pin assignments...

File: common/source/docs/common-holybro-kakuteh7-v2.rst (hunk 2 above)
Comment (Hwurzburg) [reply common/source/docs/common-holybro-kakuteh7-v2.rst:157]: havent the foggiest....can you check...I did not create those tables...

**Pull request merged**